### PR TITLE
dummy commit

### PR DESCRIPTION
**Pull request by emgogh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Django Query Inspector
 
+TEST
```
Comment (emgogh) [inline README.md:3]: is this really necessary?

Comment (therg) [reply README.md:3]: Absolutely necessary.

Comment (emgogh) [reply README.md:3]: You make a reasoned argument.  I'll consider it.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
```
Comment (therg) [inline README.md:4]: Efficient use of white space.

Comment (emgogh) [reply README.md:4]: I'm devoutly K&R compliant.

Comment (emgogh) [reply README.md:4]: Popcorn is not a complete meal.

Comment (emgogh) [reply README.md:4]: Unless it's caramel popcorn.

**Pull request closed without merging**